Model/Data/MobileLandingConfig.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * @category    ClassyLlama
+ * @package
+ * @copyright   Copyright (c) 2018 Classy Llama Studios, LLC
+ */
+
+namespace TurnTo\SocialCommerce\Model\Data;
+
+use Magento\Framework\View\Element\Block\ArgumentInterface as BlockArgumentInterface;
+use TurnTo\SocialCommerce\Api\TurnToConfigDataProviderInterface;
+use TurnTo\SocialCommerce\Helper\Config as TurnToConfigHelper;
+use TurnTo\SocialCommerce\Helper\ConfigProviderHelper;
+
+class MobileLandingConfig implements TurnToConfigDataProviderInterface, BlockArgumentInterface
+{
+    /**
+     * @var TurnToConfigHelper
+     */
+    protected $configHelper;
+
+    /**
+     * @var ConfigProviderHelper
+     */
+    protected $configProviderHelper;
+
+    /**
+     * @param TurnToConfigHelper   $configHelper
+     * @param ConfigProviderHelper $configProviderHelper
+     */
+    public function __construct(TurnToConfigHelper $configHelper, ConfigProviderHelper $configProviderHelper)
+    {
+        $this->configHelper = $configHelper;
+        $this->configProviderHelper = $configProviderHelper;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getData()
+    {
+        $config = [
+            'siteKey' => $this->configHelper->getSiteKey(),
+            'host' => $this->configHelper->getUrlWithoutProtocol(),
+            'staticHost' => $this->configHelper->getStaticUrlWithoutProtocol(),
+            'skipCssLoad' => false,
+            'setupType' => 'mobileTT'
+        ];
+
+        $config = array_merge($config, $this->configProviderHelper->getSingleSignOnConfig());
```

Comment:
The mobile landing page should actually not have SSO contents output. See this for proof of what I'm saying: https://github.com/turnto/magento2-turnto-socialcommerce/pull/42/files#diff-7a4def7790103974be017206d5b2e92fL18

Reply:
This feature was requested and implemeted: https://github.com/turnto/magento2-turnto-socialcommerce/pull/24